src/pages/index.mdx

```diff
@@ -27,6 +27,8 @@ Whether you are a shipping carrier, an online marketplace, or another service pr
 ShipEngine Connect makes it easy for you to make your services available within each of our
 world-class e-commerce solutions and gain immediate access to a worldwide user base eager to take advantage of your services.
 
+> **V1:** The documentation for the version one ShipEngine Connect App can be found [here](https://connect-v1.shipengine.com/).
```

Comment:
"...documentation for version one of the..."?

Reply:
Updated Hemingway 🤣